guides/v2.2/rest/search-endpoint.md

```diff
@@ -0,0 +1,83 @@
+---
+group: rest-api
+title: Search with the /search endpoint
+contributor_name: comwrap GmbH
+contributor_link: https://www.comwrap.com/
+---
+
+By using `<host>/rest/<store_code>/V1/search`, you can search through the catalog. The response will come from your configured search engine (**Stores** > Settings > **Configuration** > **Catalog** > **Catalog Search** > **Search engine**).
+This means you can collect results from the search engine that is enabled. These results are the same as those returned by using the regular frontend search.
+
+Magento provides three types of request queries by default:
+
+*  quick_search_container
+*  catalog_view_container
+*  advanced_search_container
+
+All of them run against the `catalogsearch_fulltext` index.
+
+The basic usage is:
+
+```http
+GET <host>/rest/<store_code>/V1/search?searchCriteria[requestName]=quick_search_container&
```

Comment:
I tried this query, except I used `name` as the search term.  The `name` attribute is set to **Use in Search** is set to Yes. Based on the statement in line 8, I would expect the search to work. `catalog_view_container` doesn't work either, but `advanced_search_container` does.

Reply:
I think it is like this:
The `field` can be set to be `search_term` only in the `quick_search_container`. This means both `catalog_view` and `advanced` will not work in that case.
Please send the queries you use? I could check them.